### PR TITLE
Typescript app.js

### PR DESCRIPTION
**Pull request by oferb**

### CODE REVIEW

File: ts/app.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  newUrl += '&maxValue=' + maxValue;
+  newUrl += '&instrumentType=' + instrumentType;
+  window.location.href = newUrl;
+}
+
+function onRadioChange(radio) {
+  let minValuePicker = $('#minValue');
+  let maxValuePicker = $('#maxValue');
+  if (radio.value == 'auto') {
+    minValuePicker.prop("disabled", true);
```
Comment (mohammad-suliman) [inline ts/app.ts]: Want to make sure the double quotation marks (") is not used by mistake here.

File: ts/app.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    maxValuePicker.prop("disabled", true);
+  } else {
+    minValuePicker.prop("disabled", false);
+    maxValuePicker.prop("disabled", false);
+    $('#updateButton').prop("disabled", false);
+  }
+}
+
+function findMinAndMaxValues() {
+  if ($('#autoOption').prop("checked") == false) {
```
Comment (mohammad-suliman) [inline ts/app.ts]: So, " is used everywhere... this is not by mistake then...

File: ts/app.ts (hunk 1 above)
Comment (oferb) [reply ts/app.ts]: For sure a mistake :)
Thanks!

File: ts/app.ts (hunk 2 above)
Comment (oferb) [reply ts/app.ts]: A mistake also here :)

**Pull request merged**